src/vivarium_gates_nutrition_optimization/constants/results.py

```diff
@@ -52,12 +48,16 @@
 
 NON_COUNT_TEMPLATES = []
 
-POP_STATES = ("living", "dead", "tracked", "untracked")
-SEXES = ("male", "female")
-# TODO - add literals for years in the model
-YEARS = ()
-# TODO - add literals for ages in the model
-AGE_GROUPS = ()
+AGE_GROUPS = ("10_to_14",
```

Comment:
nit: formatting. You might consider adding `isort` and `black` to your conda environment and running them against files as you change them.

Reply:
good point, ty